docsrc/quickstart.rst

```diff
@@ -20,69 +20,189 @@ your OS distribution of choice and their packaging, where available.  If
+            faster connection initialization.
+
+*   Cyrus Aggregation - Murder -- configurations (these constitute a
+    set, with at least one of each required):
+    
+    *   murder-mupdate.conf
+            The Mupdate Master server; holds the canonical copy of the
+            ``mailboxes.db`` database.
+    *   murder-backend.conf
+            A backend server which holds the actual mailboxes and
+            interacts with frontend proxies and/or clients.
+    *   murder-frontend.conf
+            A frontend server which holds no mailboxes, but either
+            refers clients to the proper backend server for each
+            requests, or proxies those requests directly.
+
+*   Replication configurations (these constitute a set, with one master
+    and at least one replica required):
+
+    *   normal-master.conf
+            The master server which uses the ``sync_client`` program to
+            send mailbox updates to each replica on a rolling or
+            periodic basis.
+    *   normal-replica.conf
+            A typical replica server, which accepts updates from the
+            master.
+
+You should review each of these and then install as desired to
+``/etc/``, making changes as needed.  In particular, you'll need to set
+passwords for the various users used to authenticate between instances
+in a Murder or Replication environment.
+
+.. Note::
+    Continue with instructions in :ref:`The cyrus:mail user <basicserver_cyrus_user>`
+
 Feature overview
-################
+----------------
 
 The features (configuration options) supported in our reference
 packages are:
 
-Cyrus Server configured components
-
-    * event notification: yes
-    * gssapi:             no
-    * autocreate:         yes
-    * idled:              yes
-    * httpd:              yes
-    * kerberos V4:        no
-    * murder:             yes
-    * nntpd:              yes
-    * replication:        yes
-    * sieve:              yes
-    * calalarmd:          no
-    * jmap:               no
-    * objectstore:        no
-    * backup:             yes
-
-External dependencies:
-
-    * ldap:               yes
-    * openssl:            yes
-    * zlib:               yes
-    * pcre:               no
-    * clamav:             yes
-    * caringo:            no
-    * openio:             no
-    * nghttp2:            no
-    * brotli:             no
-    * xml2:               yes
-    * ical:               yes
-    * icu4c:              yes
-    * shapelib:           no
-
-Database support:
-
-    * mysql:              no
-    * postgresql:         no
-    * sqlite:             yes
-    * lmdb:               no
-
-Search engine:
-
-    * squat:              yes
-    * sphinx:             no
-    * xapian:             yes
-    * xapian_flavor:      vanilla
-
-Installation directories:
-    * prefix:             /usr
-    * sysconfdir:         /etc
++---------------------+-------------+
```

Comment:
@elliefm I'll change this to a list instead.  It works fine on my laptop, so maybe I've got a different version of Sphinx or something like that.